### ISSUE TITLE
[BUG] Unexpected behaviour with kirby-slides on tablets

### ISSUE THREAD

**Issue by troelslenda** [kirbydesign/designsystem#3408]
- [ ] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When interacting with the swipers left side, it works like expected. On the right side, it's likly, the touch event is propagated to kirby-page and captured as a swipe back gesture.
 

## Describe how to reproduce the bug
I have not been able to reproduce it in cookbook, but assume that it has something to do with the usage of kirby-page together with kirby-slider.
The bug appear when kirby is used in a webview. It happens on both Samsung Tab A and iPad, though less pronounced on latter.

## Which Kirby version was used?
9.3.2
 

## What was the expected behavior?
To have the same behaviour for kirby-slides regardless of the touch event's position.

## Add any screenshots
https://streamable.com/90kxnp

## Are there any additional context?

It looks like the bug is easier to reproduce in landscape than portrait.
When I attach a cable and use chrome://inspect the bug goes somewhat away and reappear the instant the cable is unplugged.

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
